sections/reset-user-password.md

```diff
@@ -0,0 +1,25 @@
+## Reset User Password
+
+### Description
+Resets [user](https://github.com/xplenty/xplenty-api-doc-v2/blob/master/resources/user.md) password.
```

Comment:
@janekcz - It should be:

`Sends [user](https://github.com/xplenty/xplenty-api-doc-v2/blob/master/resources/user.md) password reset instructions`